### ISSUE TITLE
Sample says it loads but then says empty buffer

### ISSUE THREAD

**Issue by thedeany** [orllewin/playdate_tape_looper#2]
I've tried with the bundled .pdx and also in the simulator from source, and every time I try to load either of the included samples, I get the "Sample loaded" toast but upon pressing A I get the "Empty buffer" toast. I'm not sure if I'm doing something wrong, but I can't get it to work.

Let me know if there's any specific testing you'd like me to try!

**Comment by orllewin** [comment]
That's strange, the empty buffer toast is there to catch some weirdness I was seeing when I was writing the load sample feature - but I thought it was fixed, I don't see it on my devices and all Playdates should have the same firmware. There'll be a 1.0.1 release soon with some other stuff in, will investigate this too.

**Comment by thedeany** [comment]
I did another test. I pressed B to record my own sample using the built-in mic, and pressed B to stop it. Pressing A works to start and stop playback of that recording just fine. I can also save the sample just fine.

I then exited the app and opened it again for a fresh start of the app. I loaded the recorded sample I had previously saved; that sample loaded and played correctly. I then attempted to load the Amen Break sample included in the app. Upon pressing A to play, I get about a second (or maybe less) of what sounds like loud static or white noise and then my _recorded_ sample plays. If I stop and play again from the beginning, that static noise is always there.

Are the default samples corrupted in my download or something? Odd that they'd be corrupted when compiling from source in the simulator too though...

I cloned the repo on macOS and also loaded the .pdx from the repo onto my Playdate from macOS for what it's worth.
